etc/skel/.config/hypr/config/keybinds.conf

```diff
@@ -40,53 +40,53 @@ bindd = , XF86AudioPrev, Previous track, exec, playerctl previous
 
 # ======= Screen Brightness =======
 
-bindde = , XF86MonBrightnessUp, Increases brightness 5%, exec, brightnessctl s +5%
-bindde = , XF86MonBrightnessDown, Decreases brightness 5%, exec, brightnessctl s 5%-
+bindd = , XF86MonBrightnessUp, Increases brightness 5%, exec, brightnessctl s +5%
+bindd = , XF86MonBrightnessDown, Decreases brightness 5%, exec, brightnessctl s 5%-
 bindd = $mainMod SHIFT, P, Runs the calculator application, exec, gnome-calculator
 bindd = $mainMod, L, Lock the screen, exec, swaylock-fancy -e -K -p 10 -f Hack-Regular
 bindd = $mainMod, O, Reload/restarts Waybar, exec, killall -SIGUSR2 waybar
 
 # ======= Window Actions =======
 
 ## Move window with mainMod + LMB/RMB and dragging
-binddm = $mainMod, mouse:272, Move the window towards a direction, movewindow
+bindd = $mainMod, mouse:272, Move the window towards a direction, movewindow
 ## Move window towards a direction
 bindd = $mainMod SHIFT, left, Move active window to the left, movewindow, l
 bindd = $mainMod SHIFT, right, Move active window to the right, movewindow, r
 bindd = $mainMod SHIFT, up, Move active window upwards, movewindow, u
 bindd = $mainMod SHIFT, down, Move active window downwards, movewindow, d
 ## Move focus with mainMod + arrow keys
-bindde = $mainMod, Move focus to the left, left, movefocus, l
-bindde = $mainMod, Move focus to the right, right, movefocus, r
-bindde = $mainMod, Move focus upwards, up, movefocus, u
-bindde = $mainMod, Move focus downwards, down, movefocus, d
+bindd = $mainMod, Move focus to the left, left, movefocus, l
+bindd = $mainMod, Move focus to the right, right, movefocus, r
+bindd = $mainMod, Move focus upwards, up, movefocus, u
+bindd = $mainMod, Move focus downwards, down, movefocus, d
```

Comment:
These binds are incorrectly formatted. The keys should be before the description. Can you please fix it?